cognite/client/_api/labels.py

```diff
@@ -76,7 +76,7 @@ def create(
                 >>> from cognite.client import CogniteClient
                 >>> from cognite.client.data_classes import LabelDefinition
                 >>> c = CogniteClient()
-                >>> labels = [LabelDefinition(external_id="valve"), LabelDefinition(external_id="pipe")]
+                >>> labels = [LabelDefinition(external_id="valve", name="labelName1"), LabelDefinition(external_id="pipe", name="labelname2")]
```

Comment:
```suggestion
                >>> labels = [LabelDefinition(external_id="valve", name="Valve"), LabelDefinition(external_id="pipe", name="Pipe")]
```

Since the external Id and already domain-specific, we should try to be consistent.